scripts/get_version.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2014 Google Inc. All rights reserved.
```

Comment:
Not sure if this should be 2016 since this is a "new" file or if it should be the original year?